ProjectLighthouse/Maintenance/MaintenanceJobs/CleanupXMLInjection.cs

```diff
@@ -0,0 +1,57 @@
+﻿#nullable enable
+using System.Threading.Tasks;
+using LBPUnion.ProjectLighthouse.Helpers;
+using LBPUnion.ProjectLighthouse.Types;
+using LBPUnion.ProjectLighthouse.Types.Levels;
+using LBPUnion.ProjectLighthouse.Types.Profiles;
+using LBPUnion.ProjectLighthouse.Types.Reports;
+using LBPUnion.ProjectLighthouse.Types.Reviews;
+
+namespace LBPUnion.ProjectLighthouse.Maintenance.MaintenanceJobs;
+
+public class CleanupXmlInjection : IMaintenanceJob
+{
+    private readonly Database database = new();
+    public string Name() => "Cleanup unsanitized XML";
+    public string Description() => "Sanitizes all strings in levels, reviews, comments, users, and scores";
```

Comment:
```suggestion
    public string Description() => "Sanitizes all user-generated strings in levels, reviews, comments, users, and scores to prevent XML injection. Only needs to be run once.";
```